unbored/activity.py

```diff
@@ -116,19 +116,22 @@ def action_move_down( self ) -> None:
             self.post_message( self.Moved() )
             self.scroll_visible()
 
-    def calc_accessibility( access ) -> str:
+    def calc_accessibility( self ) -> str:
+        access = self.activity.accessibility
         multiplied_number = int(access * 10)
         remaining_slots = 10 - multiplied_number
-        return '♥' * multiplied_number + 'o' * remaining_slots
+        return '[yellow]![/] ' * multiplied_number + '[red]♥[/] ' * remaining_slots
 
-    def calc_party( party ) -> str:
-        party_size = 9 if number < 10 else (10 if number == 10 else 10)
-        return '🯅' * party_size + ('+' if party_size > 10 else 'o' * (10 - party_size))
+    def calc_party( self ) -> str:
+        party = self.activity.participants
+        party_size = min(party, 10) # 9 if party > 10 else (10 if party == 10 else 10)
```

Comment:
still trying to wrap my head around this logic, but:

less than 10: show like others (color + gray)
exactly 10: show like others
more than 10: show 9 and `+` :
```
🯅 🯅 🯅 🯅 🯅 🯅 🯅 🯅 🯅 +
```